packages/@okta/vuepress-site/docs/guides/authenticators-okta-verify/main/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Okta Verify (Push/OTP) integration guide
+---
+
+<div class="oie-embedded-sdk">
+
+<ApiLifecycle access="ie" /><br>
+<ApiLifecycle access="Limited GA" /><br>
+
+<StackSelector />
+
+This guide shows you how to integrate Okta Verify into your app using the embedded SDK.
+
+---
+**Learning outcomes**
+
+* Understand the Okta Verify enrollment and challenge flows
+* Learn step-by-step how to integrate Okta Verify into your authentication use case
+
+**What you need**
+
+* <StackSnippet snippet="whatyouneedsdk" />
+* <StackSnippet snippet="whatyouneedorg" />
+
+**Sample code**
+
+* <StackSnippet snippet="samplecode" />
+
+**Related use cases**
+
+<StackSnippet snippet="relatedusecases" />
+
+---
+
+## Overview
+
+...
+
+<!--
+<div class="common-image-format">
+
+![Diagram showing the WebAuthn enrollment flow](/img/authenticators/authenticators-webauthn-flow-overview.png)
+
+</div>
+
+As the service provider, you can provide WebAuthn support to your users by enabling it in your Okta org and building out support for it in your application using the Embedding SDK.
+-->
+
+## Update configurations
+<!--
+
+Before you can start using Web Authentication (WebAuthn), create an Okta org application as described in <StackSnippet snippet="orgconfigurepwdonly" inline/>. Then add WebAuthn to your app integration by executing the following steps:
+
+### Add WebAuthn to the Okta org
+
+1. In the Admin Console, go to **Security > Authenticators**.
```

Comment:
This seems like 2 steps:
1. Launch the Admin Console for your Okta org.
1. Chose **Security > Authenticators**.